src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchForTranscript.ts

```diff
@@ -29,51 +32,88 @@ type Options = {
 };
 
 type FetchPayload = {
-  transcriptId: string;
+  genomeId: string;
   geneId: string;
+  transcriptId: string;
   options: Options;
 };
 
+type ProductGeneratingContextFragment = Pick<
+  ProductGeneratingContext,
+  'product' | 'cdna' | 'cds'
+>;
```

Comment:
don't you have it defined in `fetchSequenceChecksums.ts`?

Reply:
✅

Reply:
✅